app/uk/gov/hmrc/economiccrimelevyreturns/controllers/StartController.scala

```diff
@@ -127,6 +135,13 @@ class StartController @Inject() (
       )
     )
 
+  private def addPeriodKeyToSession(periodKey: String)(implicit
```

Comment:
Can this be a more general method on the Base controller?

Reply:
Done